.gitignore

```diff
@@ -63,3 +63,6 @@ package-lock.json
 
 # Files build by scripts
 /src/thirdparty/semver.browser.js
+
+# Temporary files for testing CLA checks
```

Comment:
What is this for?

Reply:
.tmp folder was created in the local environment to simulate the github environment for some local testing purposes.